nix build failure

```
❯ nix run github:pkulak/matui
error: builder for '/nix/store/xf9bk4amab4x7b6xhv0q07gr7nlq7wyp-matui-0.4.5.drv' failed with exit code 101;
       last 25 log lines:
       >    Compiling human_bytes v0.4.3
       >    Compiling matui v0.4.5 (/build/7mk1sk3fl7r7dqp1sd6z81a6ccnw7g0k-source)
       >     Finished `release` profile [optimized] target(s) in 7m 26s
       > Executing cargoInstallPostBuildHook
       > Finished cargoInstallPostBuildHook
       > Finished cargoBuildHook
       > buildPhase completed in 7 minutes 28 seconds
       > Running phase: checkPhase
       > Executing cargoCheckHook
       > cargoCheckHook flags: -j 4 --profile release --target x86_64-unknown-linux-gnu --offline -- --test-threads=4
       >    Compiling matui v0.4.5 (/build/7mk1sk3fl7r7dqp1sd6z81a6ccnw7g0k-source)
       > error[E0616]: field `symbol` of struct `ratatui::buffer::Cell` is private
       >    --> src/widgets/textinput.rs:301:24
       >     |
       > 301 |             .map(|c| c.symbol.clone())
       >     |                        ^^^^^^ private field
       >     |
       > help: a method `symbol` also exists, call it with parentheses
       >     |
       > 301 |             .map(|c| c.symbol().clone())
       >     |                              ++
       >
       > For more information about this error, try `rustc --explain E0616`.
       > error: could not compile `matui` (lib test) due to 1 previous error
       > warning: build failed, waiting for other jobs to finish...
       For full logs, run 'nix log /nix/store/xf9bk4amab4x7b6xhv0q07gr7nlq7wyp-matui-0.4.5.drv'.
```

--- a/src/widgets/textinput.rs
+++ b/src/widgets/textinput.rs
@@ -298,7 +298,7 @@ mod tests {
 
         buf.content()[(line * width)..((line + 1) * width)]
             .iter()
-            .map(|c| c.symbol.clone())
+            .map(|c| c.symbol().clone())
             .collect()
     }
 }

